research/existing-landscape.md

```diff
@@ -193,3 +193,65 @@ polling and persisted in memory by default
    while still being available for local flag evaluation.
  - It's possible to update the `processor` used by the client to [read from a
    file](https://docs.launchdarkly.com/sdk/features/flags-from-files).
+
+## Split
+
+The following is an example of a basic flag evaluation. Split refers to the
+response type as a treatment and it's always a string.
+
+```typescript
+  // getTreatment(splitName: string, attributes?: Attributes): Treatment
+  // getTreatment(key: SplitKey, splitName: string, attributes?: Attributes): Treatment
+  const treatment = client.getTreatment("test-user", "test-feature");
```

Comment:
You may want to show both calls. This one and the overloaded that that includes custom user attributes.